frontend/src/components/Gym.js

```diff
@@ -0,0 +1,14 @@
+import React from "react"
+
+function Gym(props) {
```

Comment:
I fear it might become confusing having views and reusable components in the same directory. Home.js is different to Gym.js in that Home.js is a view itself, meant to be used when a specific url path is detected by React Router, while Gym.js is a component that could be reused in multiple parts of the application.

can you please create a separate directory `views` at the same level where `components` is? so we have
`frontend/src/components` and `frontend/src/views`

move to the `views` directory the files like `Home.js`, `About.js`, `Login.js`, etc. Basically all that are specified in `App.js` inside the Router. Keep in mind that this will require changes in the imports of some files.